src/Microsoft.Management.Infrastructure.CimCmdlets/CimIndicationWatcher.cs

```diff
@@ -258,9 +258,6 @@ private void NewSubscriptionResultHandler(object src, CimSubscriptionEventArgs a
         /// If set EnableRaisingEvents to false, which will be ignored
         /// </para>
         /// </summary>
-#if !CORECLR
-        [BrowsableAttribute(false)]
-#endif
```

Comment:
`BrowsableAttribute` is back in .NET Core, so we should leave `[BrowsableAttribute(false)]` there.